osu.Game/Database/RealmArchiveModelImporter.cs

```diff
@@ -163,7 +163,11 @@ public async Task<IEnumerable<Live<TModel>>> Import(ProgressNotification notific
 
                 if (imported.Count > 0 && PresentImport != null)
                 {
-                    notification.CompletionText += " Click to view.";
+                    if (HumanisedModelName == "skin")
+                        notification.CompletionText += " Click to apply.";
+                    else
+                        notification.CompletionText += " Click to view.";
```

Comment:
I object to such hackery. This is possibly the worst method you could apply this in, considering that we eventually likely want this to be localisable.

If we're taking the time out to fix this then it should be fixed properly. `RealmArchiveModelImporter` should expose a `protected virtual LocalisableString GetCompletionText(int count, bool presentImport)` method, and it should be overridden in `SkinImporter` to do what you're trying to make it do.

Reply:
Yeah this a much better way. I didn't consider that the string may be localized in the future. MB